activeadmin.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'jquery-rails'
   s.add_dependency 'jquery-ui-rails'
   s.add_dependency 'kaminari',            '>= 0.15', '< 2.0'
-  s.add_dependency 'railties',            '>= 4.2', '< 5.1'
+  s.add_dependency 'railties',            '>= 4.2', '< 5.2'
```

Comment:
I don't get this diff, this change is supposed to be already in the repo. Are you sure it's properly rebased on top of latest master?